src/main/java/frc/robot/Robot.java

```diff
@@ -89,8 +136,8 @@ public void teleopInit() {
     // teleop starts running. If you want the autonomous to
     // continue until interrupted by another command, remove
     // this line or comment it out.
-    if (_autonomousCommand != null) {
-      _autonomousCommand.cancel();
+    if (mAutonomousCommand != null) {
+      mAutonomousCommand.cancel();
```

Comment:
Should this be reset to null after cancel returns?

Reply:
No, it helps that it is not reset to null so we do not have to select an autonomous path through shuffleboard every time we are testing an autonomous routine. This should not matter for competition because the Autonomous portion of the game only occurs once so this conditional will only be entered one time.